### ISSUE TITLE
Method to get value at a point X, Y

### ISSUE THREAD

**Issue by rubenjusto** [pyalot/webgl-heatmap#10]
First of all thank you for the work you have done with WebGL-HeatMap library. I am using your library for a personal project. I encounter a major barrier because the library does not have any method to obtain the intensity of a given point. 

Is there any way to get it?

**Comment by rvosa** [comment]
I second this request: how would I get the pixel data at x,y?

**Comment by pyalot** [comment]
The points are splats onto the floating point framebuffer. The intensity only exists at a specific x/y position of a pixel in that framebuffer, is that what you want?

**Comment by rvosa** [comment]
Yes, that is exactly what I want: I'd like to be able to sample specific
x,y coordinates and use them to decide whether to move towards that
location. This is for a hill-climbing demo. Could you give me a brief
pointer which buffer to query?

Op Tue, 3 Nov 2015 om 11:31 schreef Florian Bösch notifications@github.com

**Comment by pyalot** [comment]
It's possible (sometimes) to read back floating point textures. However not always. To cover those cases as well, a readback needs to perform an on-GPU conversion and read back a byte texture. This is somewhat complicated by the fact that both 32-bit and 16-bit floating point textures are supported. Furthermore there's two buffers (the front and back one) and you'd have to read the front one, they're encapsulated in node classes, and each has a texture, you can see the texture you'll want to read here: https://github.com/pyalot/webgl-heatmap/blob/master/webgl-heatmap.coffee#L627

**Comment by rvosa** [comment]
So in JavaScript that would be heatmap.heights.nodeFront.texture, correct?

**Comment by pyalot** [comment]
I think so, yeah.
